lib/robots/dor_repo/accession/technical_metadata.rb

```diff
@@ -10,12 +10,14 @@ def initialize
         end
 
         def perform(druid)
-          obj = Dor.find(druid)
-          return unless obj.is_a?(Dor::Item)
+          object_client = Dor::Services::Client.object(druid)
+          obj = object_client.find
+
+          # non-items don't generate contentMetadata
+          return unless obj.is_a?(Cocina::Models::DRO)
```

Comment:
Given https://github.com/sul-dlss/common-accessioning/pull/537, are you open to changing this to `return unless obj.dro?` ?

Reply:
Sure.